src/components/groups/group.tsx

```diff
@@ -0,0 +1,67 @@
+import { Slot, component$, JSXOutput } from "@builder.io/qwik";
+import ArrowDown from "/public/images/svg/portfolio/arrowDown.svg?jsx";
+import EditIcon from "/public/images/svg/portfolio/edit.svg?jsx";
+import { Structure, StructureBalance } from "~/interface/structure/Structure";
+import { Token } from "~/components/groups/token";
+import { formatTokenBalance } from "~/utils/formatBalances/formatTokenBalance";
+import { chainIdToNetworkName } from "~/utils/chains";
+
+export interface GroupProps {
+  createdStructure: Structure;
+}
+
+function extractData(createdStructure: Structure): JSXOutput[] {
+  const extractedArray: {
+    walletName: string;
+    symbol: string;
+    quantity: string;
+    networkName: string;
+    value: string;
+  }[] = [];
+
+  createdStructure.structureBalance.forEach(
+    (balanceEntry: StructureBalance) => {
+      extractedArray.push({
+        walletName: balanceEntry.wallet.name,
+        networkName:
+          chainIdToNetworkName[balanceEntry.wallet.chainId.toString()],
+        symbol: balanceEntry.balance.symbol,
+        quantity: formatTokenBalance(
+          balanceEntry.balance.balance,
+          balanceEntry.balance.decimals,
+        ),
+        value: balanceEntry.balance.balanceValueUSD,
+      });
+    },
+  );
+
+  return extractedArray.map((entry: any) => (
+    <Token
+      icon={`/images/svg/tokens/${entry.symbol}.svg`}
```

Comment:
entry.symbol is all uppercase, needs to be converted
`entry.symbol.toLowerCase()`